Internal mysqld error

Issue is after running the rancher master for roughly 2 weeks the master container has produced this error, this causes the container to be in a constant state of restarting.
```
150829  7:35:46 [ERROR] Incorrect integer value: '1000000000000000000000000000000000000000000000000000'
150829  7:35:46 [Warning] option 'max_connections': unsigned value 0 adjusted to 1
150829  7:35:46 [ERROR] /usr/sbin/mysqld: Error while setting value '1000000000000000000000000000000000000000000000000000' to 'max_connections'
150829  7:35:46 [ERROR] Aborting
```
Upon restarting the container it will continue to add zeros to the connections integer value.

The error before this happens says 
```
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x00000000ee315000, 109740032, 0) failed; error='Cannot allocate memory' (errno=12)
```

The host this is ubuntu14.04 on Hyper-V with a static 1GB of RAM
The master manages 2 environments 
Env1: 1x Ubuntu docker host; 1x rancherOS host (both local network)
Env2: 1x Ubuntu docker host (digital ocean)

--- a/server/artifacts/mysql.sh
+++ b/server/artifacts/mysql.sh
@@ -55,7 +55,7 @@ init_new_data_dir()
 config_mysql()
 {
     sed -i 's/^\(bind-address.*\)$/#\1/' /etc/mysql/my.cnf
-    sed -i 's/^#\(max_connections.*\)/\1/;s/100/1000/' /etc/mysql/my.cnf
+    sed -i 's/^#\(max_connections.*\)/\1/;s/100$/1000/' /etc/mysql/my.cnf
     sed -i 's/^key_buffer[[:space:]]/key_buffer_size/' /etc/mysql/my.cnf
 
     # setup to be a master

